<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>803</width>
    <height>598</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>803</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;
font: 63 14pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dateTo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:darkgreen;
font: 63 14pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dateFrom">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:darkgreen;
font: 63 14pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="validAgence">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(153, 251, 245);
background-color:rgb(84, 123, 127);
font: 28pt &quot;Niagara Engraved&quot;;</string>
    </property>
    <property name="text">
     <string>Valider: Par volum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="validAgent">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(34, 67, 53);
background-color:rgb(84, 123, 127);
font: 20pt &quot;Niagara Engraved&quot;;

</string>
    </property>
    <property name="text">
     <string>Valider: Ã©valuation par l'agent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(34, 67, 53);
font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
</string>
    </property>
    <property name="text">
     <string>Choose Agence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(34, 67, 53);
font: 18pt &quot;MS Reference Sans Serif&quot;;

</string>
    </property>
    <property name="text">
     <string>To :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(34, 67, 53);
font: 22pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Choose Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(34, 67, 53);
font: 18pt &quot;MS Reference Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>From :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="evalGains">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(153, 251, 245);
background-color:rgb(84, 123, 127);
font: 28pt &quot;Niagara Engraved&quot;;</string>
    </property>
    <property name="text">
     <string>Valider: Par Gains</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>540</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 85, 127);</string>
    </property>
    <property name="text">
     <string>in progress</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
